docs/user_guide/datasets.rst

```diff
@@ -0,0 +1,333 @@
+about fairness issues with popular benchmarking datasets, the Boston Housing 
+data has been phased out of many libraries. We migrated the dataset to  
+fairlearn after it was phased out of scikit-learn in June 2020. The dataset  
+remains in fairlearn as an example of how system racism can occur in data  
+and to show the effect of fairlearn's assessment and pre-/in-/post-processing  
+tools on real, problematic data. 
+
+**We wrote this blog post to achieve the following goals:**
+  * Educate users about the history of the dataset and its bias
+  * Show users how bias manifests in the data and in downstream modelling tasks
+  * Suggest best practices for dealing with the Boston Housing data and 
+  alternative benchmarking datasets
+
+.. _dataset_origin:
+
+Dataset Origin and Use
+^^^^^^^^^^^^^^^^^^^^^
+
+Contrary to expectation, the Boston Housing dataset was not developed for 
+economic purposes. `Harrison and Rubenfield (1978) <https://deepblue.lib.umich.edu/bitstream/handle/2027.42/22636/0000186.pdf?sequence=1&isAllowed=y>` 
+developed the dataset to illustrate the issues with using housing market data 
+to measure consumer willingness to pay for clean air. The authors use a 
+`hedonic pricing <https://www.investopedia.com/terms/h/hedonicpricing.asp>` 
+approach, which assumes that the price of a good or service can be modeled as a 
+function of features both internal and external to the good or service. The 
+input to this model was a dataset comprising the Boston Standard Metropolitan 
+Statistical Area, with the *NOX* variable serving as a proxy for air quality. 
+The paper sought to estimate the median value of owner-occupied homes (now 
+*MEDV*), and included the remaining variables to measure other neighborhood 
+characteristics. Further, the authors took the derivative of their housing 
+value equation with respect to *NOX* to measure the "amount of money households
+were willing to pay with respect to air polution levels in their census 
+tracts". The variables in the dataset are representative of the early 1970s 
+and come from a mixture of surveys, administrative records, and other research
+papers. While the paper does define each variable and suggest its impact on 
+the housing value equation, it lacks reasoning for including that particular
+set of variables.
+
+Modern machine learning practicioners have used the Boston Housing dataset as 
+a benchmark to assess the performance of emerging supervised learning 
+techniques. It's featured in `scipy lectures <https://scipy-lectures.org/packages/scikit-learn/auto_examples/plot_boston_prediction.html>`, 
+indexed in the `University of California-Irvine Machine Learning Repository 
+<https://archive.ics.uci.edu/ml/machine-learning-databases/housing/>` and in 
+Carnegie Mellon University's `StatLib <http://lib.stat.cmu.edu/datasets/boston>`, 
+and for a time was included as one of `scikit-learn's <https://scikit-learn.org/stable/modules/generated/sklearn.datasets.load_boston.html>`
```

Comment:
We may have to link to a previous version, pre-1.0, since they removed it now!

Reply:
I just removed the link here, as it this source is referenced elsewhere in the post.